kernelci/data.py

```diff
@@ -0,0 +1,21 @@
+import json
```

Comment:
Missing copyright and license statement.

Reply:
Corrected